tests/ea/mutation/mutator/fixtures/random_smarts.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import stk
+from ..case_data import CaseData
+
+
+def has_bromo(building_block):
+    fg, = building_block.get_functional_groups(0)
+    return fg.__class__ is stk.Bromo
```

Comment:
isinstance does not provide the same guarantees, because it will allow sublcasses to pass, wheres comparing it to Bromo will only work for Bromo  but not for any subclass of Bromo